Assets/Mirror/Runtime/NetworkManager.cs

```diff
@@ -489,7 +492,6 @@ void StartHostClient()
             }
 
             server.ActivateHostScene();
-            RegisterClientMessages();
 
             // ConnectLocalServer needs to be called AFTER RegisterClientMessages
```

Comment:
What about this?

Reply:
got replaced by the setHandlers.   This whole thing will go away soon once I move RegisterHandlers to the connection.
This is so convoluted right now.

Reply:
So, it is not an issue anymore if it is called before ConnectLocalServer?

Reply:
yeap,  no issue anymore